lib/gp-hmac.js

```diff
@@ -98,18 +99,18 @@ GaasHmac.prototype.apply = function(obj, authorizations) {
                  obj.url + this.SEP + 
                  dateString + this.SEP +
                  bodyString;
-  //if(this.VERBOSE)  console.log('hmacText = <<' + hmacText + '>>');
+  // if(this.VERBOSE)  console.log('hmacText = <<' + hmacText + '>>');
   //var hmacBuffer = new Buffer(hmacText, this.ENC);
   
   var hmacHash = crypto.createHmac(this.HMAC_SHA1_ALGORITHM, this.secretBuffer  )
                             .update(hmacText, 'utf8')
                             .digest('base64');
-  //if(this.VERBOSE)  console.log('hmacHash = ' + hmacHash );
+  // if(this.VERBOSE)  console.log('hmacHash = ' + hmacHash );
   var hmacHeader = this.AUTH_SCHEME + ' ' + this.user + ':' + hmacHash;
-  //if(this.VERBOSE)  console.log('hmacHeader = ' + hmacHeader);
+  // if(this.VERBOSE)  console.log('hmacHeader = ' + hmacHeader);
   obj.headers.Authorization = hmacHeader;
   obj.headers.Date = dateString;
-  return true;
+  return obj;
```

Comment:
☝️ this is actually the one line change required for changing an authorization into a requestInterceptor